src/test/java/org/openrewrite/java/jackson/UpdateSerializationInclusionConfigurationTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2025 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.jackson;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.DocumentExample;
+import org.openrewrite.InMemoryExecutionContext;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+class UpdateSerializationInclusionConfigurationTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.recipe(new UpdateSerializationInclusionConfiguration())
+          .parser(JavaParser.fromJavaVersion().classpathFromResources(new InMemoryExecutionContext(),
+            "jackson-annotations-2", "jackson-core-2", "jackson-databind-2"));
+    }
+
+    @DocumentExample
+    @Test
+    void updateSerializationInclusionOnBuilder() {
+        rewriteRun(
+          // language=java
+          java(
+            """
+              import com.fasterxml.jackson.annotation.JsonInclude;
+              import com.fasterxml.jackson.databind.json.JsonMapper;
+
+              class Test {
+                  void configure() {
+                      JsonMapper.builder()
+                        .serializationInclusion(JsonInclude.Include.NON_NULL)
+                        .build();
+                  }
+              }
+              """,
+            """
+              import com.fasterxml.jackson.annotation.JsonInclude;
+              import com.fasterxml.jackson.databind.json.JsonMapper;
```

Comment:
I've another question: in jackson3, there's no such type `com.fasterxml.jackson.databind.json.JsonMapper` (package has been changed to `tool.jackson...`).

And in jackson2, there's no method `changeDefaultPropertyInclusion(..)` on `JsonMapper`.

So modified code won't compile. Is this expected? 
Because we expect another recipe will change the package?